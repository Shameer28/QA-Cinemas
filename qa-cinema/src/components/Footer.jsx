const Footer = () => {
    return (
        <div>
            <h3>&copy; QA ByteMe</h3>
        </div>
     );
}
 
export default Footer;