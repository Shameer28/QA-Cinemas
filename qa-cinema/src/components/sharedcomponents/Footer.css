.main-footer {
    color: white;
    background-color: #212529;
    padding-top: 3em;
    position: relative;
    bottom: 0;
    width: 100%;
  }
  .center{
    display: table-cell;
  vertical-align: middle;
  }
  .link{
    text-decoration: none;
    color: white;
  }

  .link:hover{
      color: white;
  }