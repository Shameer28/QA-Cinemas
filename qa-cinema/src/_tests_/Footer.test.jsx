import Footer from '../components/sharedcomponents/Footers'
import { create } from 'react-test-renderer';


describe(`Footer testing`, () => {

    it(`Should match the snapshot`, () => {
        const instance = create(<Footer/>).toJSON();
        expect(instance).toMatchSnapshot();
    })
})