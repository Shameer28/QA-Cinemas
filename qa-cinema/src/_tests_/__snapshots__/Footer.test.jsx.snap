// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing Should match the snapshot 1`] = `
<div
  className="main-footer"
>
  <div
    className="container"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col"
        >
          <section
            className="container"
          >
            <div
              className=""
            >
              <h4>
                QA Cinema
              </h4>
              <ul
                className="list-unstyled"
              >
                <li>
                  8, St James's Bldg
                </li>
                <li>
                  61-95 Oxford St
                </li>
                <li>
                  Manchester, M1 6FQ
                </li>
                <li>
                  +44 (0)345 074 7995
                </li>
              </ul>
            </div>
          </section>
        </div>
        <div
          className="col"
        >
          <section
            className="container"
          >
            <div
              className=""
            >
              <h4>
                Links
              </h4>
              <ul
                className="list-unstyled"
              >
                <li>
                  <a
                    className="link"
                    href="/"
                  >
                    Home Page
                  </a>
                </li>
                <li>
                  <a
                    className="link"
                    href="/aboutpage"
                  >
                    About US
                  </a>
                </li>
                <li>
                  <a
                    className="link"
                    href="/contactpage"
                  >
                    Contact US
                  </a>
                </li>
              </ul>
            </div>
          </section>
        </div>
        <div
          className="col"
        >
          <section
            className="container"
          >
            <div
              className="center"
            >
              <h4>
                Legal
              </h4>
              <ul
                className="list-unstyled"
              >
                <li>
                  Terms & conditions
                </li>
                <li>
                  Accessibility
                </li>
                <li>
                  Allergens and Nutrition
                </li>
              </ul>
            </div>
          </section>
        </div>
        <div
          className="col"
        >
          <section
            className="container"
          >
            <div
              className="center"
            >
              <h4>
                Socials
              </h4>
              <ul
                className="list-unstyled"
              >
                <li>
                  <a
                    className="link"
                    href="https://github.com/Shameer28/QA-Cinemas"
                  >
                    <a
                      aria-label="github"
                      className="social-icon"
                      style={
                        Object {
                          "display": "inline-block",
                          "height": "50px",
                          "overflow": "hidden",
                          "position": "relative",
                          "verticalAlign": "middle",
                          "width": "50px",
                        }
                      }
                    >
                      <div
                        className="social-container"
                        style={
                          Object {
                            "height": "100%",
                            "left": 0,
                            "position": "absolute",
                            "top": 0,
                            "width": "100%",
                          }
                        }
                      >
                        <svg
                          className="social-svg"
                          style={
                            Object {
                              "borderRadius": "50%",
                              "fillRule": "evenodd",
                              "height": "100%",
                              "left": 0,
                              "position": "absolute",
                              "top": 0,
                              "width": "100%",
                            }
                          }
                          viewBox="0 0 64 64"
                        >
                          <g
                            className="social-svg-background"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "transparent",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <circle
                              cx="32"
                              cy="32"
                              r="31"
                            />
                          </g>
                          <g
                            className="social-svg-icon"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "transparent",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <path
                              d="M32,16c-8.8,0-16,7.2-16,16c0,7.1,4.6,13.1,10.9,15.2 c0.8,0.1,1.1-0.3,1.1-0.8c0-0.4,0-1.4,0-2.7c-4.5,1-5.4-2.1-5.4-2.1c-0.7-1.8-1.8-2.3-1.8-2.3c-1.5-1,0.1-1,0.1-1 c1.6,0.1,2.5,1.6,2.5,1.6c1.4,2.4,3.7,1.7,4.7,1.3c0.1-1,0.6-1.7,1-2.1c-3.6-0.4-7.3-1.8-7.3-7.9c0-1.7,0.6-3.2,1.6-4.3 c-0.2-0.4-0.7-2,0.2-4.2c0,0,1.3-0.4,4.4,1.6c1.3-0.4,2.6-0.5,4-0.5c1.4,0,2.7,0.2,4,0.5c3.1-2.1,4.4-1.6,4.4-1.6 c0.9,2.2,0.3,3.8,0.2,4.2c1,1.1,1.6,2.5,1.6,4.3c0,6.1-3.7,7.5-7.3,7.9c0.6,0.5,1.1,1.5,1.1,3c0,2.1,0,3.9,0,4.4 c0,0.4,0.3,0.9,1.1,0.8C43.4,45.1,48,39.1,48,32C48,23.2,40.8,16,32,16z"
                            />
                          </g>
                          <g
                            className="social-svg-mask"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "#808080",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <path
                              d="M0,0v64h64V0H0z M37.1,47.2c-0.8,0.2-1.1-0.3-1.1-0.8c0-0.5,0-2.3,0-4.4c0-1.5-0.5-2.5-1.1-3 c3.6-0.4,7.3-1.7,7.3-7.9c0-1.7-0.6-3.2-1.6-4.3c0.2-0.4,0.7-2-0.2-4.2c0,0-1.3-0.4-4.4,1.6c-1.3-0.4-2.6-0.5-4-0.5 c-1.4,0-2.7,0.2-4,0.5c-3.1-2.1-4.4-1.6-4.4-1.6c-0.9,2.2-0.3,3.8-0.2,4.2c-1,1.1-1.6,2.5-1.6,4.3c0,6.1,3.7,7.5,7.3,7.9 c-0.5,0.4-0.9,1.1-1,2.1c-0.9,0.4-3.2,1.1-4.7-1.3c0,0-0.8-1.5-2.5-1.6c0,0-1.6,0-0.1,1c0,0,1,0.5,1.8,2.3c0,0,0.9,3.1,5.4,2.1 c0,1.3,0,2.3,0,2.7c0,0.4-0.3,0.9-1.1,0.8C20.6,45.1,16,39.1,16,32c0-8.8,7.2-16,16-16c8.8,0,16,7.2,16,16 C48,39.1,43.4,45.1,37.1,47.2z"
                            />
                          </g>
                        </svg>
                      </div>
                    </a>
                    Github
                  </a>
                </li>
                <li>
                  <a
                    className="link"
                    href="https://twitter.com/"
                  >
                    <a
                      aria-label="twitter"
                      className="social-icon"
                      style={
                        Object {
                          "display": "inline-block",
                          "height": "50px",
                          "overflow": "hidden",
                          "position": "relative",
                          "verticalAlign": "middle",
                          "width": "50px",
                        }
                      }
                    >
                      <div
                        className="social-container"
                        style={
                          Object {
                            "height": "100%",
                            "left": 0,
                            "position": "absolute",
                            "top": 0,
                            "width": "100%",
                          }
                        }
                      >
                        <svg
                          className="social-svg"
                          style={
                            Object {
                              "borderRadius": "50%",
                              "fillRule": "evenodd",
                              "height": "100%",
                              "left": 0,
                              "position": "absolute",
                              "top": 0,
                              "width": "100%",
                            }
                          }
                          viewBox="0 0 64 64"
                        >
                          <g
                            className="social-svg-background"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "transparent",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <circle
                              cx="32"
                              cy="32"
                              r="31"
                            />
                          </g>
                          <g
                            className="social-svg-icon"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "transparent",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <path
                              d="M48,22.1c-1.2,0.5-2.4,0.9-3.8,1c1.4-0.8,2.4-2.1,2.9-3.6c-1.3,0.8-2.7,1.3-4.2,1.6 C41.7,19.8,40,19,38.2,19c-3.6,0-6.6,2.9-6.6,6.6c0,0.5,0.1,1,0.2,1.5c-5.5-0.3-10.3-2.9-13.5-6.9c-0.6,1-0.9,2.1-0.9,3.3 c0,2.3,1.2,4.3,2.9,5.5c-1.1,0-2.1-0.3-3-0.8c0,0,0,0.1,0,0.1c0,3.2,2.3,5.8,5.3,6.4c-0.6,0.1-1.1,0.2-1.7,0.2c-0.4,0-0.8,0-1.2-0.1 c0.8,2.6,3.3,4.5,6.1,4.6c-2.2,1.8-5.1,2.8-8.2,2.8c-0.5,0-1.1,0-1.6-0.1c2.9,1.9,6.4,2.9,10.1,2.9c12.1,0,18.7-10,18.7-18.7 c0-0.3,0-0.6,0-0.8C46,24.5,47.1,23.4,48,22.1z"
                            />
                          </g>
                          <g
                            className="social-svg-mask"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "#0099ff",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <path
                              d="M0,0v64h64V0H0z M44.7,25.5c0,0.3,0,0.6,0,0.8C44.7,35,38.1,45,26.1,45c-3.7,0-7.2-1.1-10.1-2.9 c0.5,0.1,1,0.1,1.6,0.1c3.1,0,5.9-1,8.2-2.8c-2.9-0.1-5.3-2-6.1-4.6c0.4,0.1,0.8,0.1,1.2,0.1c0.6,0,1.2-0.1,1.7-0.2 c-3-0.6-5.3-3.3-5.3-6.4c0,0,0-0.1,0-0.1c0.9,0.5,1.9,0.8,3,0.8c-1.8-1.2-2.9-3.2-2.9-5.5c0-1.2,0.3-2.3,0.9-3.3 c3.2,4,8.1,6.6,13.5,6.9c-0.1-0.5-0.2-1-0.2-1.5c0-3.6,2.9-6.6,6.6-6.6c1.9,0,3.6,0.8,4.8,2.1c1.5-0.3,2.9-0.8,4.2-1.6 c-0.5,1.5-1.5,2.8-2.9,3.6c1.3-0.2,2.6-0.5,3.8-1C47.1,23.4,46,24.5,44.7,25.5z"
                            />
                          </g>
                        </svg>
                      </div>
                    </a>
<<<<<<< HEAD
                     Twitter
                  </a>
                </li>
                <li>
                  <a
                    className="link"
                    href="https://www.facebook.com/"
                  >
=======
                     Twitter
                  </li>
                </a>
                <a
                  class="link"
                  href="https://www.facebook.com/"
                >
                  <li>
>>>>>>> 536f0aa650161d56821d1ab74d31dd8c9485b4ac
                    <a
                      aria-label="facebook"
                      className="social-icon"
                      style={
                        Object {
                          "display": "inline-block",
                          "height": "50px",
                          "overflow": "hidden",
                          "position": "relative",
                          "verticalAlign": "middle",
                          "width": "50px",
                        }
                      }
                    >
                      <div
                        className="social-container"
                        style={
                          Object {
                            "height": "100%",
                            "left": 0,
                            "position": "absolute",
                            "top": 0,
                            "width": "100%",
                          }
                        }
                      >
                        <svg
                          className="social-svg"
                          style={
                            Object {
                              "borderRadius": "50%",
                              "fillRule": "evenodd",
                              "height": "100%",
                              "left": 0,
                              "position": "absolute",
                              "top": 0,
                              "width": "100%",
                            }
                          }
                          viewBox="0 0 64 64"
                        >
                          <g
                            className="social-svg-background"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "transparent",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <circle
                              cx="32"
                              cy="32"
                              r="31"
                            />
                          </g>
                          <g
                            className="social-svg-icon"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "transparent",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <path
                              d="M34.1,47V33.3h4.6l0.7-5.3h-5.3v-3.4c0-1.5,0.4-2.6,2.6-2.6l2.8,0v-4.8c-0.5-0.1-2.2-0.2-4.1-0.2 c-4.1,0-6.9,2.5-6.9,7V28H24v5.3h4.6V47H34.1z"
                            />
                          </g>
                          <g
                            className="social-svg-mask"
                            style={
                              Object {
                                "MozTransition": "fill 170ms ease-in-out",
                                "OTransition": "fill 170ms ease-in-out",
                                "WebkitTransition": "fill 170ms ease-in-out",
                                "fill": "#0066ff",
                                "msTransition": "fill 170ms ease-in-out",
                                "transition": "fill 170ms ease-in-out",
                              }
                            }
                          >
                            <path
                              d="M0,0v64h64V0H0z M39.6,22l-2.8,0c-2.2,0-2.6,1.1-2.6,2.6V28h5.3l-0.7,5.3h-4.6V47h-5.5V33.3H24V28h4.6V24 c0-4.6,2.8-7,6.9-7c2,0,3.6,0.1,4.1,0.2V22z"
                            />
                          </g>
                        </svg>
                      </div>
                    </a>
<<<<<<< HEAD
                    Facebook
                  </a>
                </li>
              </ul>
=======
                     Facebook
                  </li>
                </a>
              </ui>
>>>>>>> 536f0aa650161d56821d1ab74d31dd8c9485b4ac
            </div>
          </section>
        </div>
      </div>
      <hr />
      <div
        className="row"
      >
        <div
          className="center"
        >
          <p
            className="col-sm"
          >
            ©
            2021
             QA Cinema | All rights reserved | Terms Of Service | Privacy
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
